spec/takeaway_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "takeaway"
+
+describe Takeaway do
+  it "can make a new instance of Takeaway" do
+    expect(Takeaway.new).to be_an_instance_of(Takeaway)
+  end
+
+  it "can respond to Takeaway_dishes method" do
+    takeaway = Takeaway.new
+    expect(takeaway).to respond_to(:order_dishes).with(2).arguments
+  end
+
+  it "can record an order" do
+    takeaway = Takeaway.new
+    takeaway.order_dishes("Avocado On Toast", 2)
+    expect(takeaway.show_order).to eq([{ name: "Avocado On Toast", price: 5.50 }, { name: "Avocado On Toast", price: 5.50 }])
+  end
+
+  it "checks order total" do
+    takeaway = Takeaway.new
+    takeaway.order_dishes("Avocado On Toast", 2)
+    takeaway.order_dishes("Jam On Toast")
+    expect(takeaway.order_total).to eq(15.5)
+  end
+
+  it "sends text after placing order" do
+    takeaway = Takeaway.new
+    takeaway.order_dishes("Avocado On Toast", 2)
+    takeaway.stub(:place_order) { "Order placed. Text sent." }
+    expect(takeaway.place_order).to eq "Order placed. Text sent."
+  end
```

Comment:
Not sure if I get your question right regarding mocking Anish, but when you start working with mocks first its usual to feel like you're 'cheating' the behaviour. The important bit here that you have to mock/stub the behaviour of the object that your object is depended on. 
You set up the scenario for the order to be placed, but in the future when your test is run, you wont be sure how to Twillio API behaves. Maybe it will be down at some point and if your Takeaway objects tests fail, you'd think your code is wrong. This is why when your tests include dependencies, you want to make sure you mock/stub the necessary behaviour that helps your object (in this case Takeaway) to be tested.